### PR TITLE
DHP-1087 Update copy on the completion screen and remove “Publish” button

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
https://sagebionetworks.jira.com/browse/DHP-1087

Updated: Changed the copy. Now looks like this:

Screenshot 2023-10-12 at 12 00 29 PM

### CODE REVIEW

File: src/components/surveys/survey-design/question-edit/rhs-subcontrols/Completion.tsx [hunk 1]
```diff
@@ -27,9 +27,10 @@ const Completion: React.FunctionComponent<{
       <StyledText>
         Customize the Completion Screen that participants will see after they've submitted their responses. <br />
         <br />
-        Surveys will be submitted on the screen prior to this Completion screen.
+        Surveys will be submitted on the screen prior to this Completion Screen.
       </StyledText>{' '}
       <Divider sx={{color: '#DFE2E6', margin: theme.spacing(4, -6, 4, -6)}} />
+      {/* TODO: syoung 10/11/2023 Figure out the copy for survey access settings and that this means to design.
```
Comment (Erin-Mounts) [inline src/components/surveys/survey-design/question-edit/rhs-subcontrols/Completion.tsx:33]:
```suggestion
      {/* TODO: syoung 10/11/2023 Figure out the copy for survey access settings and what this means to design.
```
?

**Review by Erin-Mounts — APPROVED** [review]
one nit/question on a TODO comment, otherwise looks good

**Review by hallieswan — APPROVED** [review]
Looks good to me!

File: src/components/surveys/survey-design/question-edit/rhs-subcontrols/Completion.tsx [hunk 2]
```diff
@@ -49,21 +44,17 @@ const Completion: React.FunctionComponent<{
...
-        startIcon={<FileUploadTwoToneIcon />}
-        onClick={() => alert('TODO')}
-        sx={{padding: theme.spacing(1, 2.5), margin: theme.spacing(1.5, 0, 5, 0)}}>
-        Publish Survey
-      </Button>
-      <Button> </Button>
+      <StyledText> 
+        A survey is published when it is used in a live study. 
+        Once published, it is locked and cannot be changed.
+      </StyledText>
```
Comment (hallieswan) [inline src/components/surveys/survey-design/question-edit/rhs-subcontrols/Completion.tsx:56]: Nice, this seems pretty clear to me!

**Review by hallieswan — APPROVED** [review]

**Review by camptma1 — APPROVED** [review]
approved

**Pull request merged**